src/toolkits/object_detection/object_detector.hpp

```diff
@@ -207,6 +214,8 @@ class EXPORT object_detector: public ml_model_base {
   flex_int get_training_iterations() const;
   flex_int get_num_classes() const;
 
+  variant_type convert_map_to_types(variant_map_type& result_map, std::string output_type);
```

Comment:
Could this be a static method of the class? (Or a file-local global helper function in the .cpp file?) I don't think this should depend on any instance data members